<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LeftList</class>
 <widget class="QWidget" name="LeftList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>272</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>252</width>
        <height>536</height>
       </rect>
      </property>
      <widget class="QListWidget" name="list1">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>241</width>
         <height>181</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_my_music">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>200</y>
         <width>231</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>我的音乐</string>
       </property>
      </widget>
      <widget class="QListWidget" name="list2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>220</y>
         <width>241</width>
         <height>181</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>410</y>
         <width>171</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>创建歌单</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>410</y>
         <width>31</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
      <widget class="QListWidget" name="list3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>440</y>
         <width>231</width>
         <height>91</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
